docs/linking.md

```diff
@@ -328,7 +328,7 @@ The `Promise` will reject on Android if it was impossible to check if the URL ca
 >     <queries>
 >         <intent>
 >             <action android:name="android.intent.action.VIEW" />
->             <data android:scheme="https"/>
+>             <data android:scheme="https" android:host="*" />
```

Comment:
I'm not entirely sure about your suggestion but this should potentially be better:

```suggestion
>             <data android:scheme="https" android:host="www.example.com" />
```

Otherwise the app will catch all the `"android.intent.action.VIEW"` intents for all the URLs.

Reply:
Exactly, I need to catch all URLs as I want to be able to open any link with the external browser not only one specific domain.

Reply:
> as I want to be able to open any link with the external browser not only one specific domain

Could you please elaborate? Is this specific to your use case?

Reply:
No, it's not specific to my use case. I believe this is the normal case: you want Linking to let you open any link with the external browser. My specific use case: we're a messenger like a corporate WhatsApp. Our users share links in their messages and they should be able to open them using the system browser.